### ISSUE TITLE
[CORE] Taunts in PvP

### ISSUE THREAD

**Issue by haephaistoss** [Stormforge-gg/Bugtracker-Fordring#1721]
[//]: # (Enclose links to things related to the bug using http://wowhead.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Zone] A Vision of the Past)
[//]: # ([NPC] Thoralius the Wise)
[//]: # ([Spell][Class] Frostfire Bolt)
[//]: # ([Talent][Class] Brain Freeze)
[//]: # ([Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([Npc][Drop] Thoralius the Wise)
[//]: # ([Web] Armory doesnt work)

**Description**: Taunts should cause combat, and make sitting characters stand up, thus interrupting drinking / eating.

https://youtu.be/Pe_4Z64Z2Is - Put both if you in combat

https://youtu.be/TxEkG9EfLJc - Taunt makes you stand up

https://www.warcraftmovies.com/movieview.php?id=115963

16:17 clearly visible

Taunt doesn't make you stop first aiding in testing on TBC classic ptr.

Both were recorded on TBC ptr blizz
some examples

![Screenshot_20220715-022139_Brave](https://user-images.githubusercontent.com/67190649/179131831-713000f7-268d-49c6-a69f-582dc90e8eb1.png)

https://www.ownedcore.com/forums/world-of-warcraft/world-of-warcraft-exploits/209384-dark-command-trick-dk.html#post1367850

**Current behaviour**:

**Expected behaviour**:

**Comment by ScottieDOESknow** [comment]
![image](https://user-images.githubusercontent.com/43949261/179177900-dd633941-28ec-45dc-a745-e696895a5b70.png)
source https://wowwiki-archive.fandom.com/wiki/Hand_of_Reckoning

![image](https://user-images.githubusercontent.com/43949261/179180097-40f5b8cc-b38c-462f-8d67-d5cb40aaa4c7.png)
source https://urbad.net/blue/eu/10453475326-0608_Recent_InGame_Fixes

![image](https://user-images.githubusercontent.com/43949261/179180268-5a5abf20-f504-4794-a7b8-37273c40b356.png)
source https://wow.allakhazam.com/story.html?story=19184

![image](https://user-images.githubusercontent.com/43949261/179180887-8a705c35-eef0-4220-87da-f08a61aa8c5a.png)
source https://www.ownedcore.com/forums/world-of-warcraft/world-of-warcraft-guides/293565-ret-pve-few-bosses-dmg-boost-ca-500-1000dps.html

**Comment by haephaistoss** [comment]
yes you found the hotfix that prevented hand of reckoning from dealing dmg to players.
nice job

**Comment by ScottieDOESknow** [comment]
Not one source mentions the damage component 

'no longer able to be used' 

you cast and it does nothing

BTW these all post date your sources

**Comment by haephaistoss** [comment]
it does the same thing every taunt does. dont start your shitfest again.

yes everybody knows hand of reckoning was bugged in it's first Iteration and dealt dmg to players, which was unintended and was hotfixed, thank you karen

**Comment by haephaistoss** [comment]
https://www.mmo-champion.com/threads/645823-PvP-Hand-of-Reckoning


deals dmg no more, but it still causes combat....

I'm starting to be sure you're deliberately here to fuck the server up and troll testers

**Comment by ScottieDOESknow** [comment]
you just linked something from 2009-04-25

all my sources are from 4+ months later

**Comment by haephaistoss** [comment]
not starting shitfest again

**Comment by haephaistoss** [comment]
I give you one more chance to get the jest:

Taunt is a hit, but players are immune to it. Immune hits, like all avoided hits should still count as hits, and cause combat.

It's not the taunt part that matters, it's the immune - avoided hit part.

If you are in bubble and you get a frostbolt or something taht you return immune result to, you should still get in combat.

It has nothing to do with the taunt effect itself, and everything with how immunity and combat timer works.

If you use a spell and miss it, it should still cause combat. Immunity is t he same basically. Taunts are used to make healers stop drinking from range, or prevent restealth for the same reason - and the only benefit they do have over fearie fire, or icy touch or shomething is that they are instant and don't trigger gcd, so you can react quickly.

**Comment by haephaistoss** [comment]
https://youtu.be/Pe_4Z64Z2Is  - Put both if you in combat

https://youtu.be/TxEkG9EfLJc - Taunt makes you stand up

Taunt doesn't make you stop first aiding in testing on TBC classic ptr.

Both videos were recorded just now on blizz tbc ptr

Now you can kindly fuck off with your bs

**Comment by ScottieDOESknow** [comment]
I'm sure if nothing changed from TBC to WotLK your argument would stand.
Unfortunately a LOT changed and spells were patched and hotfixed all throughout WotLK even very late.

![image](https://user-images.githubusercontent.com/43949261/179202162-1c612c0d-3a72-43a6-8b21-ee6ef3e88ea9.png)

**Comment by haephaistoss** [comment]
so far all 3 major taunt spells ahve been proven to work, so i dont have to deal with you anymore

**Comment by haephaistoss** [comment]
https://www.warcraftmovies.com/movieview.php?id=115963

16:17

you see taunt, no dmg, says immune, puts both of them in combat, eeriely similar to tbc, seems like it hasn't changed at all, so another one of your false claims disproven

**Comment by ScottieDOESknow** [comment]
Unfortunately, that movie is from Season 5.
The hotfix below was post season 7.
Four day days after patch 3.2 to be exact.

![image](https://user-images.githubusercontent.com/43949261/179307315-3cc5cc35-5a25-4ecc-b0c6-6f0d7ce2995e.png)

Hey at least you linked a top Ret and fortunately Hoff has few movies for you to find the evidence you need from Season 8+
![back-to-work](https://user-images.githubusercontent.com/43949261/179308506-15f5e3b6-9ad7-4936-b4b4-8a80243a586b.gif)
